src/components/clusters/Clusters/Clusters.test.tsx

```diff
@@ -0,0 +1,264 @@
+import React from 'react';
+
+import { screen, withState } from '~/testUtils';
+
+import { Clusters } from './Clusters';
+
+// Mock child components to simplify testing
+jest.mock('~/components/clusters/ClusterListMultiRegion', () => ({
+  __esModule: true,
+  default: () => <div data-testid="list-tab">Cluster List Content</div>,
+}));
+
+jest.mock('../ClusterTransfer/ClusterTransferList', () => ({
+  __esModule: true,
+  default: () => <div data-testid="cluster-transfer-list">Cluster Transfer List Content</div>,
+}));
+
+jest.mock('./ClustersPageHeader', () => ({
+  ClustersPageHeader: () => <div data-testid="clusters-page-header">Clusters Header</div>,
+}));
+
+const mockNavigate = jest.fn();
+const mockLocation = {
+  pathname: '/clusters',
+  search: '',
+  hash: '',
+  state: null,
+  key: 'default',
+};
+
+jest.mock('~/common/routing', () => ({
+  useNavigate: () => mockNavigate,
+}));
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useLocation: () => mockLocation,
+}));
+
+describe('<Clusters />', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+    mockLocation.hash = '';
```

Comment:
nit:  is clearing the `hash` value really necessary?  will `resetAllMocks` do that for you maybe?